package.json

```
@@ -32,17 +32,17 @@
     "redirect"
   ],
   "peerDependencies": {
-    "vite": "^3.0.0"
+    "vite": "^4.0.0"
   },
   "peerDependenciesMeta": {
     "vite": {
       "optional": true
     }
   },
   "dependencies": {
-    "cloudflare-redirect-parser": "^0.1.0"
+    "cloudflare-redirect-parser": "0.1.0"
```

```suggestion
    "cloudflare-redirect-parser": "^0.1.0"
```
The caret here should be left as-is.